securedrop/journalist_app/__init__.py

```diff
@@ -89,13 +92,15 @@ def create_app(config):
 
     @app.errorhandler(CSRFError)
     def handle_csrf_error(e):
+        # type: (CSRFError) -> Response
         # render the message first to ensure it's localized.
         msg = gettext('You have been logged out due to inactivity')
         session.clear()
         flash(msg, 'error')
         return redirect(url_for('main.login'))
 
     def _handle_http_exception(error):
+        # type: (Error) -> Tuple[Any, Any]
```

Comment:
`Error` here should be `HTTPException` which is a generic exception base class in werkzeug 

The return type should be `Response`

Reply:
ah right there's a status code also returned, so this _should_ be `Tuple[Response, int]`